packages/@aws-cdk/aws-cloudtrail/README.md

```diff
@@ -92,4 +92,33 @@ trail.addLambdaEventSelector(["arn:aws:lambda"]);
 
 // Add an event selector to log data events for the provided Lambda functions.
 trail.addLambdaEventSelector([lambdaFunction.functionArn]);
+```
+
+Using the `Trail.onEvent()` API, a Cloudwatch event rule can be created that gets triggered for
+every event logged in CloudTrail.
+To only use the events that are of interest, either from a particular service, specific account or
+time range, they can be filtered down using the APIs available in `aws-events`. The following code
```

Comment:
which APIs? is `addEventPattern` the only one or are there others?

Reply:
That's the only one. I don't want to expand too much and hope the user would go to the aws-events module to learn more.

Reply:
I think that's reasonable.